test/read.test.ts

```diff
@@ -16,18 +16,22 @@ describe("commands/read", function () {
     restore();
   });
 
-  test("throws with invalid chain", async function () {
+  test("throws with invalid table name", async function () {
```

Comment:
`read` no longer uses the `--chain` flag, so I repurposed this test to check the table name validation.